Content.Server/Paper/PaperComponent.cs

```diff
@@ -16,6 +16,10 @@ public sealed partial class PaperComponent : SharedPaperComponent
     [DataField("stampedBy")]
     public List<StampDisplayInfo> StampedBy { get; set; } = new();
 
+    [DataField("canEdit")]
```

Comment:
same

Reply:
This keeps it consistent with the rest of the file